modules/fbx/fbx_document.cpp

```diff
@@ -2355,6 +2355,10 @@ Error FBXDocument::append_from_scene(Node *p_node, Ref<GLTFState> p_state, uint3
 	return ERR_UNAVAILABLE;
 }
 
+Error FBXDocument::write_to_filesystem(class Ref<class GLTFState>,class String const &) {
```

Comment:
```suggestion
Error FBXDocument::write_to_filesystem(Ref<GLTFState>, const String &) {
```

Reply:
oh good catch, I had copied that from the msvc error message